samples/import.ts.scala

```diff
@@ -0,0 +1,24 @@
+
+import scala.scalajs.js
+import js.annotation._
+import js.|
+
+package importedjs {
+
+package mod {
+
+@js.native
+@JSGlobal("mod")
+object Mod extends js.Object {
+  def f(x: Number): String = js.native
+}
+
+}
+
+@js.native
+@JSGlobal
+object Importedjs extends js.GlobalScope {
```

Comment:
Travis is not happy with that line. `Importedjs` should have `@JSGlobalScope`+`extends js.Any` instead of `@JSGlobal`+`extends js.GlobalScope`.

(it might not be related to the present PR--the bug might have existed before, in which case you might want to fix it in a separate PR first)